addons/pos_restaurant/static/src/overrides/components/ticket_screen/ticket_screen.js

```diff
@@ -124,9 +124,11 @@ patch(TicketScreen.prototype, {
     async onDoRefund() {
         const order = this.getSelectedOrder();
         if (this.pos.config.module_pos_restaurant && order && !this.pos.selectedTable) {
-            await this.pos.setTable(
-                order.table ? order.table : this.pos.models["restaurant.table"].getAll()[0]
-            );
+            const tableModel = this.pos.models["restaurant.table"];
+            const table = order.table_id || tableModel.getFirst();
+            if (table) {
```

Comment:
Can you confirm that you're automatically on the product_screen even when there is no table ?